fatemeh/CSS/CSS Size Units/style.css

```diff
@@ -0,0 +1,27 @@
+.A {
+  background-color: violet;
+  width: 300px;
+  height: 100px;
+}
+.A > .D{
+  background-color: aquamarine;
+  width: 40%;
+  height: 30%;
+}
+
+.B {
+  background-color: skyblue;
+  width: 40%;
+  height: 200px;
+}
+.C {
+  background-color: bisque;
+  font-size: 20px;
+  width: 100ex;
```

Comment:
ex is not relative to font-size, but "x" height, which is not relevant here.

Reply:
I changed ex to em.